packages/ngtools/webpack/src/ivy/plugin.ts

```diff
@@ -301,6 +298,26 @@ export class AngularWebpackPlugin {
     });
   }
 
+  private markResourceUsed(
+    normalizedResourcePath: string,
+    currentUnused: Set<string>,
+    visitedResources = new Set<string>(),
```

Comment:
This function is called for all Webpack modules without a visited set, so individual modules won't realize that they, or any of its transitive deps, could have already been visited when traversing another module.

Reply:
good call.  updated.